libsolidity/analysis/ReferencesResolver.cpp

```diff
@@ -219,7 +219,7 @@ void ReferencesResolver::operator()(yul::Identifier const& _identifier)
 {
 	solAssert(nativeLocationOf(_identifier) == originLocationOf(_identifier), "");
 
-	static set<string> suffixes{"slot", "offset", "length"};
+	static set<string> suffixes{"slot", "offset", "length", "address", "selector"};
 	string suffix;
```

Comment:
```suggestion
	string const& suffix;
```

Reply:
It can't be a `const&` without more changes as we would need to assign it in the declaration.